### PR TITLE
Fix some pre-commit configuration issues

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -15,15 +15,12 @@
         args:
         - --ignore=W503
     -   id: name-tests-test
-        files: ^test*/.+\.py$
```
Comment (asottile) [inline .pre-commit-config.yaml]: this is the default

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -15,15 +15,12 @@
...
     -   id: trailing-whitespace
     -   id: check-added-large-files
 -   repo: git://github.com/asottile/reorder_python_imports.git
     sha: 3d86483455ab5bd06cc1069fdd5ac57be5463f10
     hooks:
     -   id: reorder-python-imports
-        files: .*\.py$
```
Comment (asottile) [inline .pre-commit-config.yaml]: this is the default

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -15,15 +15,12 @@
...
-        exclude: ^(yelp/vendor/|yelp/bootstrapped/).*\.py$
```
Comment (asottile) [inline .pre-commit-config.yaml]: this is copy pasted from the *THREE MILLION LINE CODEBASE*

**Comment by ajm188** [comment]
lg2m

File: .pre-commit-config.yaml (hunk 1 above)
Comment (asottile) [inline .pre-commit-config.yaml]: oh actually, looking more closely at this, it's matching python files in a `tes` `test` or `testttttttttt` directory, heh.

Comment (ajm188) [inline .pre-commit-config.yaml]: heh, how about `files: ^tests?*/.+\.py$`?

Comment (ajm188) [inline .pre-commit-config.yaml]: err I mean `^tests?/.+\.py$`

Comment (asottile) [inline .pre-commit-config.yaml]: Or just omit it, since the default is tests/ and you use tests/

Comment (ajm188) [inline .pre-commit-config.yaml]: Sounds good to me. I did not know what the default was.

**Pull request merged**